admin/class-nginx-helper-admin.php

```diff
@@ -427,10 +427,11 @@ public function add_timestamps() {
 
 		foreach ( headers_list() as $header ) {
 			list( $key, $value ) = explode( ':', $header, 2 );
-			if ( 'Content-Type' === $key && strpos( trim( $value ), 'text/html' ) !== 0 ) {
+			$key = strtolower( $key );
+			if ( 'content-type' === $key && strpos( trim( $value ), 'text/html' ) !== 0 ) {
```

Comment:
@thrijith Just to be safer side, use `strtolower` on `$key` for comparison. Please do same below as well.